src/hooks/hookResources.ts

```diff
@@ -549,6 +556,32 @@ export function createQuestionnaireCompletionTask(
           text: 'questionnaire'
         },
         valueCanonical: `${questionnaireUrl}`
+      },
```

Comment:
This looks great, do we still need to have the questionnaire info? Is that to match the CRD support?